omero/sysadmins/unix/server-install-homebrew.txt

```diff
@@ -41,155 +51,196 @@ running::
+
+    export PATH="/usr/local/opt/python/libexec/bin:$PATH"
+
+.. note:: 
+   To keep things a little cleaner, add the following enviorment variable to your .bash_profile::
+
+    # Environment variable pointing to Homebrew Python location
+    export PYTHON_BREW=/usr/local/opt/python/libexec/bin
+
+   and append it to the :envvar:`PATH`::
+
+    export PATH="$PYTHON_BREW:$PATH"
+
+5. Check that Python is working and is version 2.7.x::
 
     $ which python
-    /usr/local/bin/python
+    /usr/local/opt/python/libexec/bin/python
+
     $ python --version
-    Python 2.7.9
+    Python 2.7.13
 
-The installation of OMERO via Homebrew depends upon two alternate
-repositories containing extra formulae:
-https://github.com/Homebrew/homebrew-science for the HDF5 formula and
-https://github.com/ome/homebrew-alt for all the OME-provided formulae
-and older versions of Ice. To add these, run::
+6. For developing with OMERO, or Python in general, we would recommend you use VirtaulEnv.
+   VirtualEnv allows us to develop python applications without having to 
+   worry about clashing thirdparty packages for different Python projects.
 
-    $ brew tap homebrew/science
-    $ brew tap ome/alt
+   Use pip to get `VirtualEnv <https://virtualenv.pypa.io/en/stable/>`__::
 
-.. note::
+    $ pip install virtualenv
+
+   With VirtualEnv installed, create a virtual enviorment::
+
+    $ virtualenv ~/Virtual/omero
+
+   This will create a folder to hold Python libraries in the the directory :file:`~Virtual/omero/lib`
+
+.. note:: You can activate the VirtualEnv enviorment that we created using::
+
+    $ source ~/Virtual/omero/bin/activate
+
+   This will swtich to using pip and python in the VirtualEnv directory 
+   :file:`~/Virtual/omero/bin` and any pip libraries you install, whilst the VirtaulEnv
+   is activated, will be installed to :file:`source ~/Virtual/omero/lib`.
+
+   .. note:: (OPTIONAL) You can add an `alias` to your .bash_profile file to make this step easier::
 
-    The Homebrew formulae used below provide Python bindings. As
-    described in `Homebrew and Python`_, you should **not** be in an
-    active virtual environment when you ``brew install`` them.
+        alias startVmOmero="source ~/Virtual/omero/bin/activate"
 
-See the :download:`step01_deps.sh <walkthrough/osx/step01_deps.sh>` script for
-the steps described above.
+      Reload bash_profile in OSX::
+
+        $ source ~/.bash_profile
+
+      Now you can activate the VirtualEnv enviorment using::
+
+        $ startVmOmero
+
+.. Note::
+   See the :download:`step01_deps.sh <walkthrough/osx/step01_deps.sh>` script for the steps described above.
 
 OMERO installation
 ------------------
 
-OMERO |release| server
-^^^^^^^^^^^^^^^^^^^^^^
+Pre-built server
+^^^^^^^^^^^^^^^^
 
-To install and deploy the |release| release of OMERO.server, run::
+1. Using the command-line Terminal, prepare a place for your OMERO server to 
```

Comment:
Not sure why terminal now has an initial capital? It didn't above. (Gets one again below though.)